content/_data/changelogs/weekly.yml

```diff
@@ -7634,6 +7634,91 @@
   # pull: 4723 (PR title: Reduce heading for repeatable)
   # pull: 4727 (PR title: Fix ItemGroupMixInTest compilation.)
 
+- version: "2.238"
+  date: 2020-05-25
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4714
+    issue: 62181
+    authors:
+    - jglick
+    message: |-
+      Fix a deadlock involving custom loggers during agent startup (regression in 2.231).
+  - type: rfe
+    category: rfe
+    pull: 4673
+    authors:
+    - sorend
+    message: |-
+      Support Bearer tokens in Jenkins-CLI <code>-auth</code> parameter.
+  - type: rfe
+    category: rfe
+    pull: 4531
+    issue: 61206
+    authors:
+    - timja
+    message: |-
+      Add system read support for 'Node Monitoring Configuration' and configuring clouds.
+  - type: rfe
+    category: rfe
+    pull: 4531
+    issue: 61206
+    authors:
+    - timja
+    message: |-
+      Add Agent/ExtendedRead support for viewing agent configuration, system information, and logs.
+  - type: rfe
+    category: rfe
+    pull: 4730
+    authors:
+    - Vlatombe
+    - jglick
+    message: |-
+      Fix a thread safety issue in <code>Computer#getLogDir</code>.
+  - type: rfe
+    category: rfe
+    pull: 4700
+    issue: 61973
+    authors:
+    - fqueiruga
+    message: |-
+      Revamp the sidebar task list with improved aesthetics and accessibility.
+  - type: rfe
+    category: rfe
+    pull: 4728
+    issue: 61458
+    authors:
+    - escoem
+    message: |-
+      Allow users with Jenkins.MANAGE permissions to reload configuration from disk.
```

Comment:
We use the UI labels when referring to permissions not the code ones

Overall/Manage

Reply:
```suggestion
      Allow users with Overall/Manage permissions to reload configuration from disk.
```